### PR TITLE
Add Dependencies on Install

### PR DESCRIPTION
**Pull request by TosinJs**
## Description

https://www.loom.com/share/bc81a196a7644b76ace0bc283bc2ed33

## How Has This Been Tested?

**Screen recording (compulsory)**:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Improvement (general improvements to how code looks/works)

## Checklist

- [ ] I have created a branch for this patch/feature.
- [ ] My pull request addresses exactly one patch/feature.
- [ ] Each individual commit in the pull request is meaningful.
- [ ] I have tested manually on my deployed review app.
- [ ] My code follows the code style of this project.

- [ ] I have requested and received approval from a peer
- [ ] I have requested and received (initial) approval from the code owner
- [ ] I have completed QA testing
- [ ] I have received final approval from the code owner

### CODE REVIEW

**Review by timigod — CHANGES_REQUESTED** [review]
Let's check out other projects and see how they do it.

File: README.md [hunk 1]
```diff
@@ -188,6 +188,30 @@ Returns
...
+requests==2.28.2
+```
+
+#### poetry
+
+If you are managing your project with Poetry, execute the following command to add these dependencies to your pyproject.toml file:
+
+```shell
+poetry add ecdsa==0.19.0 httpx==0.27.0 pydantic==2.7.4 pydantic_core==2.18.4 requests==2.28.2
+```
```
Comment (Bashorun97) [inline README.md:213]: I don't think there's a need for requirements.txt if dependencies are being managed with poetry.

You can also add `pip install -r requirements.txt` as an alternative command if the user manages dependencies with pip.

**Comment by TosinJs** [comment]
> Let's check out other projects and see how they do it.

@timigod I have gone through some popular packages and it looks like the packages only give instructions on how to install their packages ( pip install ). I did not come across any packages that auto-update the dependencies for the users. 

https://www.loom.com/share/f7f35c8452a54bc49af5bb9d3af21bb6

File: README.md (hunk 1 above)
Comment (TosinJs) [reply README.md:213]: @Bashorun97 thanks for the review. I added the Poetry instructions for situations where users of the package are using Poetry for their dependency management.  This way they know what packages they need to install in their project after generating the Sauron files.

**Pull request merged**